docs/GlobalAPI.md

```diff
@@ -524,7 +590,13 @@ Use `test.concurrent.each` if you keep duplicating the same test with different
 #### 1. `test.concurrent.each(table)(name, fn, timeout)`
 
 - `table`: `Array` of Arrays with the arguments that are passed into the test `fn` for each row.
```

Comment:
I would keep this inline as well:

```suggestion
- `table`: `Array` of Arrays with the arguments that are passed into the test `fn` for each row. If you pass in a 1D array of primitives, internally it will be mapped to a table i.e. `[1, 2, 3] -> [[1], [2], [3]]`.
```